src/app/parcels/fetchParcelTableData.ts

```diff
@@ -159,37 +168,49 @@ export const getParcelIds = async (
     }, []);
 };
 
+export const getParcelsByIds = async (
+    supabase: Supabase,
+    filters: Filter<ParcelsTableRow, any>[],
+    sortState: SortState<ParcelsTableRow>,
+    parcelIds: string[]
+): Promise<ParcelsTableRow[]> => {
+    let query = supabase.from("parcels_plus").select("*");
+
+    filters.forEach((filter) => {
+        if (filter.methodConfig.paginationType === PaginationType.Server) {
+            query = filter.methodConfig.method(query, filter.state);
+        }
+    });
+
+    if (
+        sortState.sortEnabled &&
+        sortState.column.sortMethodConfig?.paginationType === PaginationType.Server
+    ) {
+        query = sortState.column.sortMethodConfig.method(query, sortState.sortDirection);
+    } else {
+        query = query.order("packing_datetime", { ascending: false });
+    }
```

Comment:
these lines are exactly the same in 3 places (getParcelsByIds, getParcelProcessingData & getParcelIds). shall we commonise this?